lizmap/www/css/map.css

```diff
@@ -127,7 +136,7 @@ CSS Sprite positions calculated via http://instantsprite.com/ or https://draeton
 #mapmenu .nav-list > li.map.active .icon,
 #mapmenu .nav-list > li.map a:hover .icon {
   background-position: -40px -60px;
-}
+}li.geololi.geolo
```

Comment:
I think you should revert this change...

Reply:
ups